build.sbt

```diff
@@ -27,24 +27,19 @@ val HtmlUnitVersion = "2.35.1"
 val PhantomJsDriverVersion = "1.4.4"
 val MockitoVersion = "2.18.3"
 val CssParserVersion = "1.4.0"
-val ScalatestVersion = "3.0.8-RC5"
-
-def previousVersion(scalaVer: String): Option[String] = if(scalaVer.equals(scala213)) None else Some("4.0.0")
+val ScalatestVersion = "3.0.8"
 
 lazy val mimaSettings = Seq(
-  mimaPreviousArtifacts := previousVersion(scalaVersion.value)
-    .map(v => Set(organization.value %% name.value % v))
-    .getOrElse(Set.empty)
+  mimaPreviousArtifacts := {
+    if(scalaVersion.value.equals(scala213))  Set.empty // TODO: update to 5.0.0 once released
+    else  Set(organization.value %% name.value % "4.0.0")
+  }
 )
 
-resolvers ++= DefaultOptions.resolvers(snapshot = true)
```

Comment:
These won't be necessary for Play 2.8.0 next milestone since it should not depend on Akka HTTP snapshots anymore.